Solutions/Malware Protection Essentials/Analytic Rules/BackupDeletionDetected.yaml

```diff
@@ -0,0 +1,94 @@
+id: 259de2c1-c546-4c6d-a17c-df639722f4d7
+name: Detect Malicious Usage of Recovery Tools to Delete Backup Files
+description: |
+  This analytic rule detects usage of recovery tools vssadmin, wbadmin, wmic and bcedit to delete backup files or change recovery configuration. Adversaries may use these tools to delete shadow copies and backup files to prevent recovery of files.
+  https://attack.mitre.org/techniques/T1490/
+severity: High
+status: Available 
+tags:
+  - Schema: _ASim_ProcessEvent
+    SchemaVersion: 0.1.4
+requiredDataConnectors:
+  - connectorId: CrowdStrikeFalconEndpointProtection
+    dataTypes: 
+      - CommonSecurityLog
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - SecurityAlert
+  - connectorId: SentinelOne
+    dataTypes:
+      - SentinelOne_CL
+  - connectorId: VMwareCarbonBlack
+    dataTypes:
+      - CarbonBlackNotifications_CL
+  - connectorId: CiscoSecureEndpoint
+    dataTypes:
+      - CiscoSecureEndpoint_CL
+  - connectorId: TrendMicroApexOne
```

Comment:
Kindly consider addition of new trend micro AMA data connector id as well.

Reply:
Please consider this in all activity and hunting rules

Reply:
Added TrendMicroApexAma connector in all hunting queries and analytic rules.